R/combined.R

```diff
@@ -442,3 +442,50 @@ spim_add_forecast_region <- function(samples, forecast_days) {
   samples
 
 }
+
+## This is adapted from mcstate::pmcmc_predict
+spim_pmcmc_predict <- function(object, steps, prepend_trajectories = FALSE,
```

Comment:
Probably this should be supported in mcstate too, but I won't get around to it today